File: sample/src/main/java/me/xiaopan/sketchsample/fragment/Base64ImageTestFragment.java
package me.xiaopan.sketchsample.fragment;

import android.os.Bundle;
import android.text.Editable;
import android.text.TextWatcher;
import android.view.View;
import android.widget.EditText;

import butterknife.BindView;
import me.xiaopan.sketchsample.AssetImage;
import me.xiaopan.sketchsample.BaseFragment;
import me.xiaopan.sketchsample.BindContentView;
import me.xiaopan.sketchsample.R;
import me.xiaopan.sketchsample.widget.SampleImageView;

@BindContentView(R.layout.fragment_base64_test)
public class Base64ImageTestFragment extends BaseFragment {

    @BindView(R.id.edit_base64TestFragment)
    EditText editText;

    @BindView(R.id.image_base64TestFragment)
    SampleImageView imageView;

    @Override
    public void onViewCreated(View view, Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        editText.addTextChangedListener(new TextWatcher() {
            @Override
            public void beforeTextChanged(CharSequence s, int start, int count, int after) {

            }

            @Override
            public void onTextChanged(CharSequence s, int start, int before, int count) {
                showImage(s.toString());
            }

            @Override
            public void afterTextChanged(Editable s) {
            }
        });

        editText.setText(AssetImage.BASE64_IMAGE);
    }

    private void showImage(String imageText) {
        imageView.displayImage(imageText);
    }
}

File: sample/src/main/java/me/xiaopan/sketchsample/fragment/Base64ImageTestFragment.kt
package me.xiaopan.sketchsample.fragment

import android.os.Bundle
import android.text.Editable
import android.text.TextWatcher
import android.view.View
import android.widget.EditText
import me.xiaopan.sketchsample.AssetImage
import me.xiaopan.sketchsample.BaseFragment
import me.xiaopan.sketchsample.BindContentView
import me.xiaopan.sketchsample.R
import me.xiaopan.sketchsample.widget.SampleImageView
import me.xiaopan.ssvt.bindView

@BindContentView(R.layout.fragment_base64_test)
class Base64ImageTestFragment : BaseFragment() {

    val editText: EditText by bindView(R.id.edit_base64TestFragment)
    val imageView: SampleImageView by bindView(R.id.image_base64TestFragment)

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        editText.addTextChangedListener(object : TextWatcher {
            override fun beforeTextChanged(s: CharSequence, start: Int, count: Int, after: Int) {

            }

            override fun onTextChanged(s: CharSequence, start: Int, before: Int, count: Int) {
                showImage(s.toString())
            }

            override fun afterTextChanged(s: Editable) {}
        })

        editText.setText(AssetImage.URI_TEST_BASE64)
    }

    private fun showImage(imageText: String) {
        imageView.displayImage(imageText)
    }
}
